smarty/templates/main.tpl

```diff
@@ -478,7 +478,7 @@
                     Powered by LORIS &copy; {$currentyear}. All rights reserved.
                 </div>
       		<div align="center" colspan="1">
-                    Created by <a href="http://mcin-cnim.ca/" target="_blank">
+                    Created by <a href="http://mcin-cnim.ca/" target="_blank" rel="noopener noreferrer">
```

Comment:
These (in particular noreferrer) shouldn't be added to hosts which we know we control like mcin.ca or loris.ca

Reply:
Does this include `<a href="http://www.mni.mcgill.ca" target="_blank"> Montreal Neurological Institute and Hospital </a>`  as well ??

`

Reply:
Depends how much you trust McGill security. :)